packages/core/src/components/card/card.scss

```diff
@@ -26,6 +26,10 @@
   color: card.$heading-anchor-color;
   display: flex;
 
+  &:visited {
+    color: card.$heading-anchor-color;
```

Comment:
Is `color: inherit` hier dan ook niet knapper?

Reply:
Die pakte niet volgens mij, wss omdat `color` van zichzelf al een inheriting property is.

Reply:
`:visited` is een state, en die kan niet ‘erven’ van hetzelfde element zonder die state.

Reply:
^^ dat is nou typisch een denkfout die veroorzaakt wordt door de SASS nesting-syntax; daardoor lijkt het net alsof iets een child is; in dit geval is die dat niet.

Reply:
Ah interessant. Ik werd eerder op het verkeerde been gezet door de psuedoclasses `::before` waar het wel werkt.

Reply:
Yep, dat is _wel_ een descendant.